Reference/Security/Cookies/index.md

```diff
@@ -0,0 +1,25 @@
+---
+versionFrom: 9.0.0
+---
+
+# Cookies
+
+We use cookies to understand the usage of the website, improve the website performance, and to increase the relevance of our communication and advertising.
```

Comment:
I don't think this applies to Umbraco and definitely not to our communication and advertising, as these cookies are set on the domain you're hosting your own Umbraco installation on.

We should primarily document what cookies are set and for what reason (basically only for accessing the back-office), so users know whether they need to include these in their own cookie policy.